internal/it/ring_buffer.go

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright (c) 2008-2022, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License")
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package it
+
+import (
+	"context"
+	"testing"
+
+	"go.uber.org/goleak"
+
+	hz "github.com/hazelcast/hazelcast-go-client"
+)
+
+func RingbufferTester(t *testing.T, f func(t *testing.T, rb *hz.Ringbuffer)) {
+	name := func() string {
```

Comment:
How about `makeName` instead of `name`?

Reply:
sounds better.
c2324b27434bf80e44fbaf7d5bab6e20ad5c490e